### PR TITLE
Add Always On on Replit

### PR DESCRIPTION
**Pull request by kaaax0815**
Get the Repl Url
Create a Account on Uptimerobot
Insert the URL as HTTP
Your Bot stays on forever

Source: https://repl.it/talk/learn/Hosting-discordjs-bots-on-replit-Works-for-both-discordjs-and-Eris/11027
Issue: https://github.com/SudhanPlayz/Discord-MusicBot/issues/74

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require("dotenv").config();//Loading .env
+const keepAlive = require('./server');
```
Comment (kaaax0815) [inline index.js:2]: load module from file

File: index.js [hunk 2]
```diff
@@ -34,4 +35,5 @@ fs.readdir("./commands/", (err, files) => {
 });
 
 //Logging in to discord
+keepAlive();
```
Comment (kaaax0815) [inline index.js:38]: starting server

File: package.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "@discordjs/opus": "^0.3.2",
     "discord.js": "^12.3.1",
     "dotenv": "^8.2.0",
+    "express": "^4.17.1",
```
Comment (kaaax0815) [inline package.json:17]: Express is the server, so you have to install it

**Comment by SudhanPlayz** [comment]
We dont need the express server for now so yea will do on v4 cuz v4 has a dashboard for admins of the bot 😁

**Pull request closed without merging**